plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -15,7 +15,7 @@
 #include "fu-uefi-bgrt.h"
 #include "fu-uefi-bootmgr.h"
 #include "fu-uefi-common.h"
-#include "fu-uefi-device.h"
+#include "fu-uefi-cod-device.h"
```

Comment:
@superm1 maybe fu-uefi-disk-device.h might be clearer? Or more unclear perhaps?

Reply:
cod is fine